common/rya.api/src/main/java/org/apache/rya/api/instance/ConfigurationFields.java

```diff
@@ -31,5 +31,7 @@
     static final String USE_FREETEXT = "sc.use_freetext";
     static final String USE_TEMPORAL = "sc.use_temporal";
     static final String USE_ENTITY = "sc.use_entity";
-    static final String USE_PCJ = "sc.use_pcj";
```

Comment:
i don't think we want to support that -- the client should be consistent with whatever is defined in rya details, or with the understood system level config (in legacy systems where rya details is not available, the thought is everyone should be interacting through a single service end point/layer)